web_src/js/features/repo-diff.js

```diff
@@ -103,4 +103,27 @@ export function initRepoDiffShowMore() {
       $('#diff-incomplete').replaceWith($(resp).find('#diff-file-boxes').children());
     });
   });
+  $(document).on('click', 'a.diff-show-more-button', (e) => {
+    e.preventDefault();
+    const $target = $(e.target);
+
+    if ($target.hasClass('disabled')) {
+      return;
+    }
+
+    $target.addClass('disabled');
+
+    const url = $target.data('href');
+    $.ajax({
+      type: 'GET',
+      url,
+    }).done((resp) => {
+      if (!resp || resp.html === '' || resp.empty) {
```

Comment:
Why do we need to check `!resp || resp.html === '' || resp.empty` here?

There is `$(resp)` later, so I suppose the `resp` is  a string? If it is so, then `if (resp)` is enough.